3rdparty/CMakeLists.txt

```diff
@@ -222,30 +218,44 @@ ADD_SUBDIRECTORY(toml11 EXCLUDE_FROM_ALL)
 SET(SPDLOG_ENABLE_PCH ON CACHE BOOL "" FORCE)
 # Not yet supported by github runners
 # SET(SPDLOG_USE_STD_FORMAT ON CACHE BOOL "" FORCE)
+SET(SPDLOG_BUILD_PIC  ON CACHE BOOL "" FORCE)
```

Comment:
Could you elaborate on this?

I don't mind it, but i'm curious to know your reasons.

Reply:
Honestly, I had a link error due to positionning in spdlog, I saw there was a flag, I enabled it, it fixed the error. 
Then again, I'm not sure how this all works, so if there is a preferred solution that's perfectly fine.